### PR TITLE
Add zziplib 0.13.71

### PR DESCRIPTION
**Pull request by intelligide**
Specify library name and version:  **zziplib/0.13.71**

- [x] I've read the [guidelines](https://github.com/conan-io/conan-center-index/wiki#how-to-submit-a-pull-request) for contributing.
- [x] I've followed the [PEP8](https://www.python.org/dev/peps/pep-0008/) style guides for Python code in the recipes.
- [x] I've used the [latest](https://github.com/conan-io/conan/releases/latest) Conan client version.
- [x] I've tried at least one configuration locally with the
      [conan-center hook](https://github.com/conan-io/hooks.git) activated.

Thanks to @KonradNoTantoo

### CODE REVIEW

**Comment by KonradNoTantoo** [comment]
This makes me happy : )
Cheers!

File: recipes/zziplib/all/conanfile.py [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+        cmake.definitions["BUILD_STATIC_LIBS"] = not self.options.shared
+
+        cmake.definitions["ZZIPCOMPAT"] = not tools.os_info.is_windows
+
+        cmake.definitions["ZZIPSDL"] = False
+        cmake.definitions["ZZIPBINS"] = False
+        cmake.definitions["ZZIPTEST"] = False
+        cmake.definitions["ZZIPDOCS"] = False
+
+        cmake.configure()
```
Comment (uilianries) [inline recipes/zziplib/all/conanfile.py:64]:
```suggestion
        cmake.configure(build_folder=self._build_subfolder)
```

File: recipes/zziplib/all/CMakeLists.txt [hunk 2]
```diff
@@ -0,0 +1,11 @@
+cmake_minimum_required(VERSION 2.8.12)
```
Comment (danimtb) [inline recipes/zziplib/all/CMakeLists.txt:1]: `CMAKE_WINDOWS_EXPORT_ALL_SYMBOLS` requires CMake 3.4 at least
```suggestion
cmake_minimum_required(VERSION 3.4)
```

File: recipes/zziplib/all/conanfile.py [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+        extracted_dir = "{}-{}".format(self.name, self.version)
+        os.rename(extracted_dir, self._source_subfolder)
+
+    def _configure_cmake(self):
+        if not self._cmake:
+            self._cmake = CMake(self)
+
+            self._cmake.definitions["BUILD_STATIC_LIBS"] = not self.options.shared
+
+            self._cmake.definitions["ZZIPCOMPAT"] = not tools.os_info.is_windows
```
Comment (danimtb) [inline recipes/zziplib/all/conanfile.py:58]: why not using here `self.settings.os == "Windows"`?

**Pull request merged**